### PR TITLE
config fixes

### PR DESCRIPTION
**Pull request by Dieterbe**
* set consumer-fetch-default to sarama default of 32768, our messages
  should really fit in that anyway.
* fix bug that was resetting consumer-fetch-default to 100

### CODE REVIEW

**Comment by Dieterbe** [comment]
in particular, the faulty `consumer-fetch-default = 100` was causing kafka to run at 100% cpu and consumption to go very slowly, both for a POC customer and in raintank-docker. this fixes it for both.

File: docs/config.md [hunk 1]
```diff
@@ -171,13 +171,13 @@ data-dir =
...
 # The default number of message bytes to fetch in a request
-consumer-fetch-default = 4096000
+consumer-fetch-default = 32768
 # The maximum amount of time the broker will wait for Consumer.Fetch.Min bytes to become available before it
 consumer-max-wait-time = 1s
 #The maximum amount of time the consumer expects a message takes to process
 consumer-max-processing-time = 1s
 # How many outstanding requests a connection is allowed to have before sending on it blocks
-consumer-fetch-default = 100
+max-open-requests = 100
```
Comment (woodsaj) [inline docs/config.md]: should this not be `net-max-open-requests`
```
inKafkaMdm.IntVar(&netMaxOpenRequests, "net-max-open-requests", 100, "How many outstanding requests a connection is allowed to have before sending on it blocks")
```

Comment (Dieterbe) [inline docs/config.md]: oops yes

**Comment by Dieterbe** [comment]
good to merge?

**Comment by woodsaj** [comment]
looks good to me.

**Pull request merged**